.github/workflows/build.yml

```diff
@@ -82,30 +85,15 @@ jobs:
   codeql-analyze:
```

Comment:
I think this job should be moved to separate file as we do in main repo

Reply:
I prefer having it here, we can depend on its completion for the build/push step